src/test/java/Obj/RegPageObj.java

```diff
@@ -0,0 +1,31 @@
+package Obj;
+
+import Holder.DriverHolder;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class RegPageObj extends DriverHolder {
```

Comment:
please rename the class using elements keyword to indicate that the elements are stored here